src/test/java/com/fasterxml/jackson/databind/ser/SimpleFilterProviderTest.java

```diff
@@ -0,0 +1,138 @@
+package com.fasterxml.jackson.databind.ser;
```

Comment:
Should go in package `src/test/java/com/fasterxml/jackson/databind/ser/filter`, not main level `ser`